src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Random random = new Random();

```

Comment:
Consider which variables should be local in the method and which should be class-level. If the `ColorSupplier` and `Random` instances are not used elsewhere in the class and are specific to the `getRandomBall` method, they should be local within that method.